plugins/modules/zos_copy.py

```diff
@@ -1386,8 +1457,10 @@ def get_file_record_length(file):
         current_line = src_file.readline()
 
         while current_line:
-            if len(current_line) > max_line_length:
-                max_line_length = len(current_line)
+            line_length = len(current_line.rstrip("\n\r"))
```

Comment:
@rexemin is "\n\r" intentional? If *.readline() ends at the first `\n`, will it see a `\r', not sure if this is a transposition but not familiar with such.

Reply:
Yes, it is intentional, the method interprets the argument as a list of characters that will be removed from the end of the string. So if it finds \n, \r, \r\n or even \n\r, it will remove them.

Reply:
Had a feeling you were going to say this :)